src/commands/sites/sites-create-template.js

```diff
@@ -0,0 +1,251 @@
+    slug: 'netlify-templates/nextjs-blog-theme',
+  },
+  {
+    name: 'Next.js Starter',
+    sourceCodeUrl: 'https://github.com/netlify-templates/next-netlify-starter',
+    slug: 'netlify-templates/next-netlify-starter',
+  },
+  {
+    name: 'Nuxt.js Starter',
+    sourceCodeUrl: 'https://github.com/Gomah/bluise',
+    slug: 'Gomah/bluise',
+  },
+  {
+    name: 'Hugo Blog',
+    sourceCodeUrl: 'https://github.com/netlify-templates/one-click-hugo-cms',
+    slug: 'netlify-templates/one-click-hugo-cms',
+  },
+]
+
+/**
+ * The sites:create-template command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const sitesCreate = async (options, command) => {
+  const { api } = command.netlify
+
+  await command.authenticate()
+
+  console.log(
+    `Choose one of our starter templates. Netlify will create a new repo for this template in your GitHub account.`,
+  )
+
+  const templateUrl = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'templateName',
+      message: 'Template:',
+      choices: templates.map((template) => ({
+        value: template.slug,
+        name: template.name,
+      })),
+    },
+  ])
+
+  const accounts = await api.listAccountsForUser()
+
+  let { accountSlug } = options
+  if (!accountSlug) {
+    const { accountSlug: accountSlugInput } = await inquirer.prompt([
+      {
+        type: 'list',
+        name: 'accountSlug',
+        message: 'Team:',
+        choices: accounts.map((account) => ({
+          value: account.slug,
+          name: account.name,
+        })),
+      },
+    ])
+    accountSlug = accountSlugInput
+  }
+
+  const { name: nameFlag } = options
+  let user
+  let site
+
+  // Allow the user to reenter site name if selected one isn't available
+  const inputSiteName = async (name) => {
+    let siteSuggestion
+    if (!user) user = await api.getCurrentUser()
+
+    if (!name) {
+      let { slug } = user
+      let suffix = ''
+
+      // If the user doesn't have a slug, we'll compute one. Because `full_name` is not guaranteed to be unique, we
+      // append a short randomly-generated ID to reduce the likelihood of a conflict.
+      if (!slug) {
+        slug = slugify(user.full_name || user.email)
+        suffix = `-${uuidv4().slice(0, SITE_NAME_SUGGESTION_SUFFIX_LENGTH)}`
+      }
+
+      const suggestions = [
+        `super-cool-site-by-${slug}${suffix}`,
+        `the-awesome-${slug}-site${suffix}`,
+        `${slug}-makes-great-sites${suffix}`,
+        `netlify-thinks-${slug}-is-great${suffix}`,
+        `the-great-${slug}-site${suffix}`,
+        `isnt-${slug}-awesome${suffix}`,
+      ]
+      siteSuggestion = sample(suggestions)
+
+      console.log(
+        `Choose a unique site name (e.g. ${siteSuggestion}.netlify.app) or leave it blank for a random name. You can update the site name later.`,
+      )
+      const { name: nameInput } = await inquirer.prompt([
```

Comment:
is there a possibility to skip that input if it is optional?

Reply:
That was mostly copied from the `sites:create` command. I think in general i'd wanna give people the choice to pick a title or not?